packages/connect/src/protocol/async-iterable-story.spec.ts

```diff
@@ -194,22 +194,19 @@ describe("full story", function () {
         { value: "delta", end: false },
       ]);
     });
-    it("should throw if close is called on a writer that is already closed", function () {
-      void writer.close();
-
-      writer
-        .close()
-        .catch((e) =>
-          expect(e).toEqual(new ConnectError("cannot close, already closed"))
-        );
+    it("should allow close on a writer that is already closed", function () {
+      writer.close();
+      writer.close();
     });
     it("should throw if write is called on a writer that is closed", function () {
-      void writer.close();
+      writer.close();
 
       writer
         .write({ value: "alpha", end: false })
         .catch((e) =>
-          expect(e).toEqual(new ConnectError("cannot write, already closed"))
+          expect(e).toEqual(
+            new Error("cannot write, WritableIterable already closed")
+          )
```

Comment:
Changed it to a regular `Error` vs a `ConnectError` as this is a generic `WritableIterable` and doesn't really have anything do with connect not to mention the error is also not at all connect related.